.github/workflows/pr-checks.yml

```
@@ -0,0 +1,77 @@
+name: PR Checks
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)